### PR TITLE
First pass at GO-CAM FAQ

### PR DESCRIPTION
**Pull request by cmungall**

### CODE REVIEW

File: _docs/faq.md [hunk 1]
```diff
@@ -1315,3 +1315,52 @@ IMEx will only call an interaction 'direct' when performed with 2 purified molec
...
 
+## What is a GO-CAM?
+
+GO-CAM stands for Gene Ontology Causal Activity Model. GO-CAMs link
+multiple GO annotations into an integrated model of a biological
+system. More information can be found on the [GO-CAM site](http://geneontology.org/cam).
+
+## What exchange format is used for GO-CAMs?
+
+GO-CAMs include more information than classic GO annotations, so
```
Comment (suzialeksander) [inline _docs/faq.md]: "_classic_" or "_standard_"?  http://geneontology.org/docs/tools-overview/ says "GO-Causal Activity Models (GO-CAMs) use a defined “grammar” for linking multiple standard GO annotations into larger models of biological function "

I don't have a preference, just think we should be consistent. @thomaspd what's the Official GO Terminology for this?

Comment (suzialeksander) [reply _docs/faq.md]: also line 1353 same question

File: _docs/faq.md [hunk 2]
```diff
@@ -1315,3 +1315,52 @@ IMEx will only call an interaction 'direct' when performed with 2 purified molec
...
+The model allows for classic GO annotations (with no causal
+connection), or for causally connected annotations. This allows for
+the capture of partial information in an incremental fashion.
+
+The GO-CAM model is simpler than that used by databases such as
+Reactome: in a GO-CAM we do not typically capture all the participants
+in a reaction, together with their stoichiometry. Instead this
+information is included in the GO term.
+
+We are currently investigation translations between pathway formats
```
Comment (suzialeksander) [inline _docs/faq.md]: investigating

File: _docs/faq.md (hunk 1 above)
Comment (thomaspd) [reply _docs/faq.md]: We used "standard" in the paper, so it's probably easiest to stick with this.

File: _docs/faq.md (hunk 2 above)
Comment (lpalbou) [reply _docs/faq.md]: I corrected some typo (e.g. investigating) and classic -> standard go annotations, but not sure it is useful to use all the time "standard" (e.g. line 1353) and sent a commit to this branch.

@cmungall "in a GO-CAM we do not typically capture all the participants
	in a reaction, together with their stoichiometry. Instead this
	information is included in the GO term."

Included in the GO term ? This seems a bit misleading to me.. True that we can differ a single GP from a complex but I don't think there is any indication of stoichiometry ?

**Pull request merged**